### DISCUSSION TITLE
nesting of CSS is() and not() on recent version

### DISCUSSION THREAD

**Discussion by shopper360mst** [beercss/beercss#237] [Q&A]
Im having trouble rolling out a site equipped with beercss. all is working well until we tested the site with some China made android where they have their own webkit stock browser namedly the brand VIVO.

to my surprise the stock browser doesnt support advance css nesting rules like 
is() 
works on single not(  ) 
but not on array of rules within e.g. not( h1, h2, h3)

I'm using npm to get the latest package and took out the css file from node_module to make some admend but there too much changes. is there a tool i can use to transpile these rules into a more normalized rule that every browser can function?

**Comment by coffeeandwork** [comment]
Maybe this one? https://lightningcss.dev/transpilation.html

**Comment by mingyuen78** [comment]
i'll give it a shot. i tried using postCSS not much luck with it as it failed to convert (too complex for it to handle)

**Comment by mingyuen78** [comment]
lightning managed to return all the dialogs and pages transitions and positioning. while looking at the css file i noticed there are a specific rule with triple hyphen namely ---gap 

so far i only know, there are double hyphen rule like --gap and --primary but what is triple hyphen for? even that is failing on the particular browser which may causes spacing errors.
